### PR TITLE
[Tier1][Cephadm][Quincy]Deployment with mix os

### PR DESCRIPTION
**Pull request by MohitBis**
[Tier1][Cephadm][Quincy]Deployment with mix os
Deploy rhcs 6.x cluster with admin node on rhel 9 and other hosts in rhel 8

### CODE REVIEW

**Comment by MohitBis** [comment]
Logs : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-HGWSF7/

File: suites/quincy/cephadm/tier-1_mix_os.yaml [hunk 1]
```diff
@@ -0,0 +1,158 @@
+#=========================================================================================================
```
Comment (vamahaja) [inline suites/quincy/cephadm/tier-1_mix_os.yaml:1]: rename to `tier-1-mix-os-deployment.yaml`

File: suites/quincy/cephadm/tier-1_mix_os.yaml [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+#   Node1 - Admin, MON, MGR, OSD, Node-Exporter, Alertmanager, Grafana, Prometheus, Crash, RGW
+#   Node2 - MON, MGR, OSD, Node-Exporter, Alertmanager, Crash
+#   Node3 - MON, MGR, OSD, Node-Exporter, Crash
+#   Node4 - MDS, RGW
+#   Node5 - Client
+#
+# Test Steps:
+#   1. Deploy cluster using cephadm with admin node on rhel 9 and other hosts in rhel 8 with all services.
+#   2. Configure Client
+#   3. Run object, block and filesystem basic operations parallelly.
```
Comment (vamahaja) [inline suites/quincy/cephadm/tier-1_mix_os.yaml:18]: Restructure description with below -

- Test Suite - Description
- Conf - Conf File Path
- Setup - Setup steps
- Test cases - Testcases covered

**Review by vamahaja — CHANGES_REQUESTED** [review]

File: conf/quincy/cephadm/tier-1_5node_mix_os.yaml [hunk 3]
```diff
@@ -0,0 +1,73 @@
+
```
[the remark this answers is not in the document]
Comment (MohitBis) [reply conf/quincy/cephadm/tier-1_5node_mix_os.yaml]: Updated

File: suites/quincy/cephadm/tier-1_mix_os.yaml [hunk 4]
```diff
@@ -0,0 +1,158 @@
+#=========================================================================================================
+# Tier-level: 1
+# Test-Suite: tier-1_mix_os.yaml
```
[the remark this answers is not in the document]
Comment (MohitBis) [reply suites/quincy/cephadm/tier-1_mix_os.yaml]: Updated

File: suites/quincy/cephadm/tier-1_mix_os.yaml (hunk 1 above)
Comment (MohitBis) [reply suites/quincy/cephadm/tier-1_mix_os.yaml]: Updated

File: suites/quincy/cephadm/tier-1_mix_os.yaml (hunk 2 above)
Comment (MohitBis) [reply suites/quincy/cephadm/tier-1_mix_os.yaml]: Updated

**Review by pranavprakash20 — APPROVED** [review]
LGTM

**Review by vamahaja — APPROVED** [review]

**Pull request merged**